src/platform/ESP32/PlatformManagerImpl.cpp

```diff
@@ -94,6 +92,9 @@ CHIP_ERROR PlatformManagerImpl::_InitChipStack(void)
         goto exit;
     }
 
+#if defined(CONFIG_RENDEZVOUS_MODE_SOFTAP) && defined(CONFIG_WIFI_AP_RANDOM_MAC)
```

Comment:
@sweetymhaiske I think `CONFIG_RENDEZVOUS_MODE_SOFTAP` is from the examples. Ideally, we should use the chosen commissioning mode (flag/variable) that is available in the sdk